### DISCUSSION TITLE
re: flask Blueprints

### DISCUSSION THREAD

**Discussion by mledezma-r7** [plangrid/flask-rebar#228] [Q&A]
Hello, dropping a quick one here

Is there any way to use flask-rebar with an existing modular app that uses flask Blueprints? Thank you

**Comment by RookieRick** [comment]
Hey there! Sorry for the late reply, I missed a notification email it seems.  
Ironically, despite having inherited the flask-rebar project I am far from a Flask expert and haven't messed with Blueprints myself, but I will ask around a bit and see if anybody has a better answer than that for you :)

**Reply by mledezma-r7** [reply]
Awesome, thank you

**Reply by RookieRick** [reply]
Hey, so as you may have guessed, I haven't found any reports of success on that front.  😞  I guess my best advice at this point would be if you have time, take a whack at it and see where things don't play nice; would be good to file Issues if we can identify what needs to change to make them play nice (extra bonus points if you come up with fixes for said issues 😄)

**Reply by jab** [reply]
In case it may be helpful, https://github.com/pallets/flask/pull/3923 was recently merged and will allow nested blueprints in the next major release of Flask.
